src/azure-cli/azure/cli/command_modules/aro/custom.py

```diff
@@ -130,7 +129,7 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
         ],
     )
 
-    sp_obj_ids = [client_sp.object_id, rp_client_sp.object_id]
+    sp_obj_ids = [client_sp, rp_client_sp]
```

Comment:
Same here. Perhaps:

```suggestion
    sp_obj_ids = [client_sp_id, rp_client_sp_id]
```

Reply:
Yes, this is incorporated in my new commit.